### PR TITLE
feat: Added a keybind for quickly setting quantity or price

### PR DESCRIPTION
**Pull request by Thunderkill**
Added a configurable keybind to quickly set the copilot suggested price or quantity of an item.

I also had to shuffle around the methods in OfferEditor and moved them to a own handler called OfferHandler

### CODE REVIEW

**Comment by Thunderkill** [comment]
There is a side effect that if you bind it to "Enter" it will first set the value and then submit it. Not sure if Jagex likes that it does 2 actions with same keypress

**Comment by cbrewitt** [comment]
> There is a side effect that if you bind it to "Enter" it will first set the value and then submit it. Not sure if Jagex likes that it does 2 actions with same keypress

Looks good overall, although the "Enter" problem does need to be addressed. As you say, this could break Jagex rules and result in the plugin being removed from the plugin hub.

File: src/main/java/com/flippingcopilot/controller/FlippingCopilotConfig.java [hunk 1]
```diff
@@ -126,4 +128,13 @@ default boolean suggestionHighlights()
     {
         return true;
     }
+    @ConfigItem(
+            keyName = "quickSetKeybind",
+            name = "Quick set keybind",
```
Comment (cbrewitt) [inline src/main/java/com/flippingcopilot/controller/FlippingCopilotConfig.java:133]: Naming this "Price/Quantity set keybind" could make it clearer to users what this does.

**Comment by Thunderkill** [comment]
> > There is a side effect that if you bind it to "Enter" it will first set the value and then submit it. Not sure if Jagex likes that it does 2 actions with same keypress
> 
> Looks good overall, although the "Enter" problem does need to be addressed. As you say, this could break Jagex rules and result in the plugin being removed from the plugin hub.

I'll just disable "Enter" keybinds

File: src/main/java/com/flippingcopilot/controller/FlippingCopilotConfig.java (hunk 1 above)
Comment (Thunderkill) [reply src/main/java/com/flippingcopilot/controller/FlippingCopilotConfig.java]: Changed

**Pull request merged**